src/components/EyesOnMe.js

```diff
@@ -1 +1,14 @@
-// Code EyesOnMe Component Here
+import React from 'react';
+
+const EyesOnMe = () => {
+  return(
+    <button
+      onFocus={ () => console.log('Good!') }
+      onBlur={ () => console.log('Hey! Eyes on me!') }
+    >
```

Comment:
It would be awesome to move these functions in their own functions outside of the return:
```suggestion
    const handleFocus = () => console.log('Good!')
    const handleBlur = () => console.log('Hey! Eyes on me!')
    
    <button
      onFocus={handleFocus}
      onBlur={handleBlur}
    >
```